### ISSUE TITLE
TensorRT crashes on converting a simple convolutional graph

### ISSUE THREAD

**Issue by ethereon** [tensorflow/tensorflow#23467]
TensorRT crashes during conversion of a trivial graph with the following error:

```
python: customWinogradConvActLayer.cpp:48:
nvinfer1::cudnn::WinogradConvActLayer::WinogradConvActLayer(const string&, const EngineTensors&, const EngineTensors&, const nvinfer1::ConvolutionParameters&, bool, const std::vector<float>&): 
Assertion `matchNbDims(inputs[0], outputs[0]) && (inputs.size() == 1 || inputs[1].extent == outputs[0].extent)' failed.
```

The following minimal example reproduces this issue:

```
import tensorflow as tf
from tensorflow.contrib import tensorrt

input = tf.placeholder(tf.float32, shape=(1, 32, 32, 3))
weights = tf.ones([3, 3, 3, 8], tf.float32)
output = tf.nn.conv2d(input, weights, strides=[1, 1, 1, 1], padding='SAME')
output = output + tf.ones((8,), dtype=tf.float32)
output = tf.nn.relu(output, name='output')

tensorrt.create_inference_graph(
    input_graph_def=tf.get_default_graph().as_graph_def(), outputs=[output.op.name])
```

I've been able to reproduce this issue on multiple machines with similar configurations.

A few things to note:

- The addition and the terminal operation are critical for the issue to manifest
- The `relu` can be replaced with most other ops (eg: `tf.identity`) and the issue still manifests

A variant of this bug was previously reported here: https://github.com/tensorflow/tensorflow/issues/22577 (and was closed without resolution).

**System information**
- OS Platform and Distribution: Ubuntu 16.04
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.11.0
- Python version: 2.7.12
- CUDA/cuDNN version: 9.0/7
- GPU model and memory: GTX 1080 Ti

**Comment by harshini-gadige** [comment]
@aaroey  PTAL

**Comment by doldre** [comment]
You may try output = tf.ones((8,), dtype=tf.float32) + output

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aaroey: It has been 17 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by aaroey** [comment]
@smit-hinsu

**Comment by smit-hinsu** [comment]
Thanks for reporting this issue. But, closing this issue to consolidate all the discussions in the original reported issue #22577.

Just posted an update there. https://github.com/tensorflow/tensorflow/issues/22577#issuecomment-447532086
